wip-0009.md

```diff
@@ -61,14 +61,38 @@ Upon reception of multiple candidates:
 2. Block candidates from active identities are prioritized over those from non-active identities.
 3. If there are still multiple candidates, the one with the lowest VRF value is chosen.
 
-With the previous changes, the newcomer mining probability is bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies.
+With the previous changes, the newcomer mining probability is bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies. Just as an illustrative example with the aforementioned network conditions, the newcomer mining probability would be almost 5%.
```

Comment:
```suggestion
With the previous changes, the newcomer mining probability is bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies. As an illustrative example using the same network conditions as before, with the new validation rules in place the newcomer mining probability would go down to almost 5%.
```

Reply:
"The previous changes" still sounds confusing to me. It's not obvious whether it's referring to the changes contained in the proposal, the referred changes that took place before these and caused the current situation, or any other former changes.